src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
```

Comment:
Avoid using variable names like 'obj' as it decreases code readability. Consider using a more descriptive name for the object.